### ISSUE TITLE
model predicts the same Chinese characters

### ISSUE THREAD

**Issue by xieyidi** [SeanNaren/deepspeech.pytorch#498]
When I use Chinese speech and corresponding Chinese translation content as data to train a model, the test results and correct speech content are as follows, several Chinese characters appear frequently, such as '的'、‘一’. What is the reason for this? ?

Model predictions： [['一的的一的'], ['一的的'], ['一的的'], ['一的的'], ['一的的'], ['一的的'], ['一的的一的'], ['一的的'], ['一的一的的'], ['一的的'], ['一的一的一的'], ['一的吕的'], ['一的蛆的'], ['一的一的的'], ['一的的']]
Voice content： [['家住焦东矿家属院的申老虎由于所在企业效益不佳加上长年患病医疗费已花去二万多元生活极度困难'], ['至于岭南文化的形成则是南越土著文化同中原文化周边文化及海洋文化融合的结果'], ['有的职员为饱私囊外出采购货物以次充优弄虚作假坑了公司也坑了边疆百姓'], ['正因为此斯诺曾感触颇深地说鲁迅虽身材瘦弱矮小但和鲁迅在一起你必须仰视着去领会那崇高的思想'], ['矿与矿采区与采区班组与班组之间的劳动竞赛热火朝天地在全局展开'], ['那天上午半阴下午放晴可是到了晚上饮酒赏月时却又浮云满天未能赏玩'], ['其种植的红富士苹果以色泽艳丽果质细脆汁多味美和极耐贮运而享誉海内外'], ['那枝杨少爷从朝鲜带回来的毛笔让小姐给扔进字纸篓里我娘一时没察觉给丢了'], ['下雨时节大家都慌忙上山去挖笋又到沟里捉田鸡无奈没有油常常吃得胃酸'], ['当他与官兵吃完除夕夜的饺子检查完油库安全措施的落实又不顾疲劳连夜下山'], ['另外外贸物资运输量将有较大的增长冶炼物资和西部原油运输量也有一定增长'], ['那水既然能够养天马为什么就不能养人严密的推理之中隐含着热烈的梦幻'], ['据报载去年某住宅区曾有女房客从高层楼宇扔下用过的月经纸天女散花般飘飘而下'], ['干革命搞建设都要有一批勇于思考勇于探索勇于创新的闯将'], ['干革命搞建设都要有一批勇于思考勇于探索勇于创新的闯将']]

**Comment by sculyi** [comment]
Does the loss decrease during training？ When I train the model on THCHS30, the loss is always about 100.0.

**Comment by xieyidi** [comment]
the loss is around 200  at the end of training

**Comment by kli017** [comment]
what dataset are you using?

**Comment by xieyidi** [comment]
thchs30

**Comment by HandsLing** [comment]
@xieyidi  maybe your label has some wrong

**Comment by raotnameh** [comment]
@xieyidi  try using Adam as an optimizer.
